pom.xml

```diff
@@ -241,7 +241,7 @@ flexible messaging model and an intuitive client API.</description>
     <!-- Set docker-java.version to the version of docker-java used in Testcontainers -->
     <docker-java.version>3.2.13</docker-java.version>
     <kerby.version>1.1.1</kerby.version>
-    <testng.version>7.3.0</testng.version>
+    <testng.version>7.5</testng.version>
```

Comment:
I learn the latest version is 7.6.1. Why do we use 7.5 here? Also, 7.5 is not of a full semantic version format.

Reply:
7.6.1 requires JDK 11 or higher.  The Pulsar-CI have a JDK 8 test.